help_to_heat/portal/download_views.py

```diff
@@ -271,7 +271,8 @@ def add_extra_row_data(referral, exclude_pii=False):
 
     if exclude_pii:
         for key in pii_keys:
-            row.pop(key)
+            if key in row.keys():
+                row.pop(key)
```

Comment:
Sanity-checking: if not `key in row.keys()` then there's no data there and we'll just end up with a blank cell anyway?

Reply:
yeah, there's the following line in the xlsx writer:
```py
    for row_num, referral_data in enumerate(rows):
        for col_num, entry in enumerate(column_headings):
            to_write = referral_data.get(entry) or ""
            worksheet.write(row_num + 1, col_num, to_write)
```
so should default to "" if not present (it's not uncommon for columns to be missing, loft access often is)